src/transformers/configuration_utils.py

```diff
@@ -250,10 +250,17 @@ def get_config_dict(
                 )
             else:
                 msg = (
-                    "Model name '{}' was not found in model name list. "
-                    "We assumed '{}' was a path, a model identifier, or url to a configuration file named {} or "
-                    "a directory containing such a file but couldn't find any such file at this path or url.".format(
-                        pretrained_model_name_or_path, config_file, CONFIG_NAME,
+                    "Can't load '{}'. Make sure that:\n\n"
+                    "- '{}' is either a model name in {}\n\n"
+                    "- or '{}' is a correct url to a community model identifier which has a '{}' file\n\n"
```

Comment:
this is not a url, but a "model identifier", like `dbmdz/bert-base-german-cased` – also it's now the preferred way to load a model so maybe put this first

Reply:
Ok I put the actual url adress to the amazon servers there:
`https://s3.amazonaws.com/models.huggingface.co/bert/dbmdz/bert-base-german-cased` instead of just `dbmdz/bert-base-german-cased`, but yes it should be more coherent to only put `dbmdz/bert-base-german-cased` in the error message - will change that!
 but that might be more